CHANGELOG.md

```diff
@@ -1,10 +1,19 @@
 # Change Log
 
+## [0.6.1](https://github.com/openeuropa/oe_theme/tree/0.6.1) (2018-08-13)
+[Full Changelog](https://github.com/openeuropa/oe_theme/compare/0.6.0...0.6.1)
+
+**Merged pull requests:**
+
+- OPENEUROPA-772: Update compose.json to have openeuropa/drupal-core-require-dev. [\#118](https://github.com/openeuropa/oe_theme/pull/118) ([voidtek](https://github.com/voidtek))
```

Comment:
I would update this message to something that explains the reasoning why this change was made, that would be more informative to the persons reading the changelog rather than than some kind of dependency was added for unknown reasons.